flink-cdc-common/src/main/java/org/apache/flink/cdc/common/schema/Schema.java

```diff
@@ -396,6 +397,9 @@ private void checkColumn(String columnName, DataType type) {
          * @param columnNames columns that form a unique primary key
          */
         public Builder primaryKey(String... columnNames) {
+            for (int i = 0; i < columnNames.length; i++) {
+                columnNames[i] = columnNames[i].toLowerCase();
```

Comment:
We should avoid modify common class as other connectors may support upper case, what about modify this in `PaimonMetaDataApplier`?

Reply:
Hi @lvyanquan, thank you for your kind suggestions.
I've modified PaimonMetaDataApplier and PaimonMetadataApplierTest accordingly, and keep common class unchanged as before. All test cases have passed, hope this may meet your expectations, many thanks!